### PR TITLE
Add 2 network (SeniorBlockCoin & RoyalSportsCity)

### PR DESCRIPTION
**Pull request by miladsoft**

### CODE REVIEW

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            CoinType = 6599,
+            PremineReward = 3000000000,
+            PoWBlockReward = 3000,
+            PoSBlockReward = 1000,
+            LastPowBlock = 25000,
+            MaxSupply = 21000000000,
+            GenesisText = "Decentralized Royal  Sports City",
+            TargetSpacing = TimeSpan.FromSeconds(64),
+            ProofOfStakeTimestampMask = 0x0000000F, // 0x0000003F // 64 sec
+            PoSVersion = 3
```
Comment (sondreb) [inline src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs:28]: If this is a new network, you should consider using PoS v4, perhaps you can give some instructions @dangershony ?

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+{
+    internal class RoyalSportsCitySetup
+    {
+        internal static RoyalSportsCitySetup Instance = new RoyalSportsCitySetup();
+
+        internal CoinSetup Setup = new CoinSetup
+        {
+            FileNamePrefix = "royalsportscity",
+            ConfigFileName = "royalsportscity.conf",
+            Magic = "01-4D-59-43",
```
Comment (sondreb) [inline src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs:18]: Do not reuse Magic values from different networks. Cybits is already using these magic bytes. The magic bytes is responsible for making sure that node of "chain x" does not connect to "chain y".

File: src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace Blockcore.Networks.SeniorBlockCoin.Networks.Policies
+{
+   /// <summary>
+   /// Blockcore sample coin-specific standard transaction definitions.
+   /// </summary>
+   public class SeniorBlockCoinStandardScriptsRegistry : StandardScriptsRegistry
+   {
+      // See MAX_OP_RETURN_RELAY in stratisX, <script.h>
+      public const int MaxOpReturnRelay = 40;
```
Comment (sondreb) [inline src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs:16]: This should probabl be "8x" something, not 40. Or do you want to support only a single signature?

**Review by sondreb — CHANGES_REQUESTED** [review]
Some feedback given, Dan should give more. Will review again later.

File: src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs (hunk 3 above)
Comment (miladsoft) [reply src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs:16]: This was the default value. I change it to 83.

Comment (sondreb) [reply src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs]: Great!

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs (hunk 1 above)
Comment (miladsoft) [reply src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs]: I changed it to 4 .

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs (hunk 2 above)
Comment (miladsoft) [reply src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs]: I changed it to "01-52-53-43"

**Review by sondreb — APPROVED** [review]
This is now OK for me, but @dangershony should review it as a final step before merging.

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            SecretAddress = 239,
+            GenesisTime = 1641376738,
+            GenesisNonce = 128636,
+            GenesisBits = 0x1F00FFFF,
+            GenesisVersion = 1,
+            GenesisReward = Money.Zero,
+            HashGenesisBlock = "00007a766fd57aef404cc1ae16d74f616074c924d41aeed23e727f9a00b83c01",
+            HashMerkleRoot = "f547b27d5390b11f578ff3b0855b9b21547bbca3b54f39820c2de371001ea71f",
+            DNS = new[] { "seedregtest.royalsportscity.com", "seedregtest.royalsportscity.net" },
+            Nodes = new[] { "130.185.120.45", "185.235.43.17" },
```
Comment (dangershony) [inline src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs:76]: this ips are probably wrong on regtest

File: src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Setup/ConversionTools.cs [hunk 5]
```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Blockcore.Networks.SeniorBlockCoin.Networks.Setup
+{
+   public class ConversionTools
```
Comment (dangershony) [inline src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Setup/ConversionTools.cs:8]: Was that added by the coin auto generator?

**Review by dangershony — APPROVED** [review]
Looks good to me, however there is lots of data, I assume most of the blockchain lower level stuff where not changed and left to the default values.

**Pull request merged**